generation/2d_diffusion_autoencoder/2d_diffusion_autoencoder_tutorial.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    w = torch.Tensor(clf.coef_).float().to(device)
</pre><p>What's this w used for? Can we included these cells in the main evaluate cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I changed it to the next cell where it's actually used</p>